src/DX12/RHI/DX12CommandList.cpp

```diff
@@ -324,12 +350,19 @@ void DX12CommandList::ClearTexture(const RHITextureBinding& binding,
                        desc.name);
             commandList->ClearRenderTargetView(binding.texture->GetOrCreateRTVDSVView(dxTextureView),
                                                clearValue.color.data(),
-                                               0,
-                                               nullptr);
+                                               dxClearRects.size(),
+                                               dxClearRects.data());
```

Comment:
Does this correctly handle the case with 0 clear rects? Since the last param is no longer nullptr, (not sure if size being 0 means dx12 ignores the data parameter)